docs/filters.md

```diff
@@ -382,6 +382,24 @@ basicAuth("/path/to/htpasswd")
 basicAuth("/path/to/htpasswd", "My Website")
 ```
 
+## webhook
+
+The `webhook` filter makes it possible to have your own authentication and
+authorization endpoint as a filter.
+
+Headers from the incoming request will be copied into the request that
+is being done to the webhook endpoint. Responses from the webhook with
+status code less than 300 will be authorized, rest unauthorized.
+
+Examples:
+
+```
+webhook("https://custom-webhook.example.org/auth")
+```
+
+The webhook timeout has a default of 2 seconds and can be globally
```

Comment:
Why not also specifying the timeout as an optional second param to the filter?

Reply:
From parameter point of view it's fine, but from client connection pool you would have per `webhook * timeout` times connection pools. We would rather like to have less connection pools, than more.

:)